pkg/scheduler/core/extender.go

```diff
@@ -255,6 +255,8 @@ func convertToNodeNameToVictims(
 // Filter based on extender implemented predicate functions. The filtered list is
 // expected to be a subset of the supplied list. failedNodesMap optionally contains
 // the list of failed nodes and failure reasons.
+// It should return error if there's cache inconsistency between default scheduler and extender
+// so that some filtered nodes are missing from nodeNameToInfo.
```

Comment:
We can just simply revise the paragraph to:

```
// Filter based on extender implemented predicate functions. The filtered list is
// expected to be a subset of the supplied list; otherwise the function returns an error.
// failedNodesMap optionally contains the list of failed nodes and failure reasons.
```

Reply:
Thanks! Adjusted and pls check.